include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -917,8 +917,8 @@ namespace VectorTools
       additional_data.mapping_update_flags = (update_values | update_JxW_values);
       MatrixFree<dim, Number> matrix_free;
       matrix_free.reinit (mapping, dof, constraints,
-                          QGauss<1>(fe_degree+1), additional_data);
-      typedef MatrixFreeOperators::MassOperator<dim, fe_degree, 1, Number> MatrixType;
+                          QGauss<1>(fe_degree+2), additional_data);
+      typedef MatrixFreeOperators::MassOperator<dim, fe_degree, fe_degree+2, 1, Number> MatrixType;
```

Comment:
@kronbichler I think this is already being addressed in @masterleinad PRs, see my comment https://github.com/dealii/dealii/pull/3262#discussion_r86342318